kybra/compiler/kybra_generate/src/generators/header.rs

```diff
@@ -0,0 +1,26 @@
+pub fn generate() -> proc_macro2::TokenStream {
```

Comment:
Please make this look like Azle with a directory and such

Reply:
Done in e2647202f3a44245d6719d104f7cb79c4f85e783